src/vscode/treeviews/connections/DatabricksConnectionManagerAzure.ts

```diff
@@ -103,7 +103,13 @@ export class DatabricksConnectionManagerAzure extends DatabricksConnectionManage
 		}
 
 		try {
-			let session: vscode.AuthenticationSession = await vscode.authentication.getSession("microsoft", scopes);
+			let options: vscode.AuthenticationGetSessionOptions = {
+				clearSessionPreference: false,
+				createIfNone: true,
```

Comment:
I would probably only specify the `createIfNone` and omit all the others as there is no need to re-set the default again.
if its only one setting then, I would do it inline in the original command:
`... await vscode.authentication.getSession("microsoft", scopes, { createIfNone: true });`